analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -448,121 +425,98 @@ def get_query(source: ObjectStream) -> ObjectStream:
 
 
 # %% [markdown]
-# ### Standalone ServiceX for subsequent `coffea` processing
+# ### Caching the queried datasets with `ServiceX`
 #
-# Using `servicex-databinder`, we can execute a query and download the output.
-# As the files are currently accessible through `rucio` only with ATLAS credentials, you need to use an ATLAS ServiceX instance to run this (for example via the UChicago coffea-casa analysis facility).
+# Using the queries created with `func_adl`, we are using `ServiceX` to read the ATLAS Open Data files to build cached files with only the specific event information as dictated by the query.
```

Comment:
> ATLAS Open Data

should be CMS

Reply:
[66ff1ba](https://github.com/iris-hep/analysis-grand-challenge/pull/103/commits/66ff1ba7404a27006989cff15e0b71847c587648)